### ISSUE TITLE
ModOrganizer 2 No Archive tab

### ISSUE THREAD

**Issue by GSDFan** [TanninOne/modorganizer#365]
I the alpha MO there is no archive tab in the right hand pane. This is with Skyrim and Fallout 4.

**Comment by TechAngel85** [comment]
Confirming this. I have no Archive tab either.

**Comment by TanninOne** [comment]
This is intentional. The ability to have mo "manage" bsas is gone in mo2 which makes the archives tab fairly useless.

**Comment by TechAngel85** [comment]
What does this mean, exactly, for MO now where BSA management is concerned?

**Comment by GrantSP** [comment]
So does this mean the so-called "dummy" plugins, ie. HighResTexturePack0*.esp, are now vital?
That being the case the flags and tooltips for these files will need to be stripped out of the code.

**Comment by TanninOne** [comment]
Yes, the dummy mods are required in mo2 and yes, I'll have to update some tooltip and such ;)

**Comment by Freso** [comment]
FWIW, the "1st run" interactive tutorial still (MO2 alpha 1) wants you to navigate to the "Archives" tab.

Also, what's the reason for disabling BSA management in MO?

**Comment by TanninOne** [comment]
The tutorial should be fixed in the alpha 2 if I didn't mess up the packaging again.
BSA management required very sinister hacks to allow more than a handful of bsas to be loaded through the ini file. And the ability to decouple asset load order from their respective esp can break a few mods (very few actually). This same problem can happen with extracting bsas which I also no longer endorse.
The fix I wrote (PMOP, also quite a lot of work) was misunderstood and frequently ignored. The step project even went out and advised people to ignore it.

**Comment by monyarm** [comment]
So now if i want to have less dummy esp's ill have to :
create a "Dummy.esp"
then rename the bsa and ba2 file to be :
Dummy - archivename.extention

say , if you can't put archive managing back in mo2 , then how about making it so that you can right click on a dummy esp and have an option for it to do the above for it's archives ?

**Comment by crazykilla15** [comment]
What is PMOP?

Also, i second the above notion. Automate the renaming of BSA's to a hard coded dummy plugin, name unchangeable for simplicity(that way it can always be X amount of characters into the bsa name, which could simplify logic determining what mods have been dummified)

**Comment by GrantSP** [comment]
PMOP == Potential Mod Order Problem
This is a plugin in MO 1.x to check for errors in your left-hand order.

**Comment by crazykilla15** [comment]
What was its intended effect? What did people think it did?

**Comment by GrantSP** [comment]
As with a lot of MO features, most people just couldn't be bothered to examine what that plugin did, which is why we had to provide this [page](http://forum.step-project.com/topic/8624-pmop-warning-messages/) to inform users what it did.

**Comment by TanninOne** [comment]
Renaming the bsas to "dummy - xyz.bsa" would remove the possibility to reorder those assets individually so the next thing people will want is a way to reorder the dummy bsas among each other. Then a way to have multiple dummy-bsa groups. Then a way to automate sorting for the respective esps.
And I will get the hate-mails why I messed up loot-sorting for those mods.

**Comment by rsagris** [comment]
I can understand removing the overall feature of managing BSA's without their associated ESP/ESM, but what are the problems with extracting the BSA's yourself and then deactivating the ESP/ESM, all the while responsibly moving this overall extracted mod within my Left-pane mod order as I see fit?

I've never heard what problem this would bring, other than those few mods that insist on checking for other mods by ESP name.

**Comment by gab** [comment]
The main issue with Potential Mod Order Problem is that if you let it reorder mods for you, it does it in a completely retarded fashion. Last time I tested it, it decided to put Dawnguard in the middle of my load order. That's the reason why the STEP team advises people against using it.

The message about the potential problem also doesn't tell you what the problem is about. I'd rather see a mod with such a problem turn red in the left pane with some indication in conflicts of which script is getting overwritten in inverse order of the esp load order. It would make fixing the problem really easy, with actual knowledge of what you're doing instead of obeying a scary mysterious message.

Here's hoping you change your mind and put back the feature but disable it by default.

(What we really need though, is to remove user control on right pane sorting, and have a load order tool that sorts the left pane, using LOOT-style hints but actually looking at the whole package content - at least .esp + scripts. After all this whole clusterfuck comes from .pex files coming separately from the .esp.)

**Comment by TechAngel85** [comment]
Tech from the STEP team here. I want to be clear here. The STEP team **_ONLY_** advises pure STEP users to ignore the PMOP messages. That is, if you're just installing the STEP Guide. Anything beyond the Guide, you'll would want to investigate the PMOP warnings. The reason for this recommendation is because we have done the work for the users to determine if the PMOP warnings produced with the STEP Guide are actually harmful or not. They are not. Installing any mods beyond the STEP Guide which causes a PMOP warning is on the user to make those determinations.

As Gab mentions, PMOP reorders the mods in the left pane based on script conflicts vs plugin order. This is good and proper advice in most cases, but it can lead to a lot of false-postives due to the simplicity of the design (_which as far as I understand can't easily be improved upon_). This can break the intended mod order for STEP which, in turn, causes the wrong assets to be loaded into the game. Anyone using PMOP should re-evaluate their left pane after running the tool because of the mod re-ordering it performs.

With this said, STEP has already created dummy plugins for the archives we include in preparation for MO2's move back to standardized BSA management rules.
